.nvmrc

```diff
@@ -0,0 +1 @@
+v12.18.2
```

Comment:
v14 is the latest LTS. If we'd bump the version we could have a longer compatibility.

Reply:
I believe 12 is the one we are targetting on.